cmd/network/verification_test.go

```diff
@@ -64,7 +64,7 @@ func Test_egressVerificationSetup(t *testing.T) {
 			name: "ClusterId optional",
 			e: &EgressVerification{
 				ClusterId:       "",
-				SubnetId:        "subnet-a",
+				SubnetId:        []string{"subnet-a"},
```

Comment:
Can you also write a test with multiple subnets?

Reply:
Yes, we definitely need more test coverage.